app/templates/etc/prod/conf.sh

```diff
@@ -0,0 +1,28 @@
+
+# Define overrides for init script defaults here, defaults shown in comments..
+
+# name of the project
+#NAME=${0##*/}
+
+# location of your project's root
+# SOURCE_DIR=/space/projects/$NAME.live
+
+# js entry point
+#SOURCE_FILE=boot.js
+
+# server script to be launched
+#SERVER_SCRIPT=./node-server/node-app.js
+
+# port at which node server should be listening
+#APP_PORT=9040
+APP_PORT=80
```

Comment:
Yes, httpd will be delivered by it as installed but not launched out of the box. But it is still being used a lot. All our team servers have it enabled, as well as all servers I touched.. ;-)